server/src/dashboards/dashboards.dto.ts

```diff
@@ -0,0 +1,30 @@
+import { IsArray, IsNumber, IsOptional, IsString } from 'class-validator'
+import { Transform } from 'class-transformer'
+import { Layout } from './dashboard.entity'
+
+export class CreateDashboardDto {
+  @IsString()
+  name: string
+
+  @IsOptional()
+  @IsString()
+  description: string
+
+  @Transform(({ value }) => Number.parseInt(value, 10))
+  @IsNumber()
+  category: number
+}
+
+export class UpdateDashboardDto {
```

Comment:
Use can use `PartialType` as it used in `settings.dto.ts`. It would automatically make all the properties optional.

Reply:
For now we can leave the already created dtos, but at least refactor the dtos created in this PR. If possible we can update the previous dtos as well.